src/product-type-export.js

```diff
@@ -246,11 +247,22 @@ export default class ProductTypeImport {
     })
   }
 
+  getProductTypeClient () {
+    const clientEndpoint = this.client.productTypes
+
+    if (this.config.where)
+      clientEndpoint.where(this.config.where)
+
+    return clientEndpoint
+  }
+
   downloadProductTypes (file) {
     const writeStream = createWriteStream(file)
     writeStream.write('[')
     let isFirst = true
-    return this.client.productTypes.process(({
+
+    return this.getProductTypeClient()
+    .process(({
```

Comment:
Hmm.. there are other places indented like this, for example:
https://github.com/sphereio/sphere-product-type-export/pull/15/files/2f4e1dc682a548d14677d9fe1eed22d1c843e561#diff-9d42f85cc847a16773c7454182538dc7R581